src/test/java/net/snowflake/client/jdbc/ResultSetIT.java

```diff
@@ -1014,4 +1014,16 @@ public void testNextNegative() throws SQLException {
       rs.close();
     }
   }
+
+  @Test
+  public void testGeographyColumnType() throws Exception {
+    try (Connection conn = init()) {
+      conn.createStatement()
+          .execute("create or replace table test1 (geog geography, geom geometry)");
+      ResultSetMetaData meta =
+          conn.createStatement().executeQuery("select * from test1").getMetaData();
```

Comment:
let's reuse statement

Reply:
Removed the test as it was already covered by another one.